cms/djangoapps/modulestore_migrator/tests/test_tasks.py

```diff
@@ -825,58 +705,6 @@ def test_migrate_component_library_source_key(self):
             "problem", result.componentversion.component.component_type.name
         )
 
-    def test_migrate_component_duplicate_content_integrity_error(self):
```

Comment:
This was using private helpers methods to create and test an internal error scenario which became impossible once we added `repeat_handling_strategy` and began consistently checking for and handling conflicts. If an IntegrityError happens in the migrator now, then it means that someone is calling private helper functions, or we have a bug in our code.